python/ngen_conf/src/ngen/config/init_config/noahowp_options.py

```diff
@@ -0,0 +1,282 @@
+from enum import Enum
+
+
+class PrecipPhaseOption(int, Enum):
+    """
+    integer :: opt_snf    ! precip_phase_option: options for determining precipitation phase
+        1 -> Jordan (1991) SNTHERM equation
+        2 -> rain-snow air temperature threshold of 2.2°C
+        3 -> rain-snow air temperature threshold of 0°C
+        4 -> precipitation phase from weather model
+        5 -> user-defined air temperature threshold
+        6 -> user-defined wet bulb temperature threshold
+        7 -> binary logistic regression model from Jennings et al. (2018)
+    source: https://github.com/NOAA-OWP/noah-owp-modular/blob/30d0f53e8c14acc4ce74018e06ff7c9410ecc13c/src/OptionsType.f90#L11-L18
+    """
+
+    sntherm = 1
+    rain_snow_air_temp_threshold_2_2_C = 2
+    rain_snow_air_temp_threshold_0_C = 3
+    precip_phase_from_weather_model = 4
+    user_defined_air_temp_threshold = 5
+    user_defined_wet_bulb_temperature_threshold = 6
+    binary_logistic_regression_model = 7
+
+
+class RunoffOption(int, Enum):
+    """
+    integer :: opt_run    ! runoff_option: options for runoff
+        1 -> TOPMODEL with groundwater (Niu et al. 2007 JGR)
+        2 -> TOPMODEL with an equilibrium water table (Niu et al. 2005 JGR)
+        3 -> original surface and subsurface runoff (free drainage)
+        4 -> BATS surface and subsurface runoff (free drainage)
+        5 -> Miguez-Macho&Fan groundwater scheme (Miguez-Macho et al. 2007 JGR; Fan et al. 2007 JGR)
+        6 -> VIC runoff
+        7 -> Xinanjiang runoff
+        8 -> Dynamic VIC runoff
+
+    source: https://github.com/NOAA-OWP/noah-owp-modular/blob/30d0f53e8c14acc4ce74018e06ff7c9410ecc13c/src/OptionsType.f90#L19-L27
+    """
+
+    topmodel_with_groundwater = 1
+    topmodel_with_equilibrium_water_table = 2
+    original_surface_and_subsurface_runoff = 3
+    bats_surface_and_subsurface_runoff = 4
+    Miguez_Macho_Fan_groundwater_scheme = 5
```

Comment:
can we use lowercase in these enums to be consistent?

Reply:
For sure. Changing that now.